lib/rtp_h264/payloader.ex

```diff
@@ -38,7 +37,7 @@ defmodule Membrane.RTP.H264.Payloader do
               ]
 
   def_input_pad :input,
-    accepted_format: %H264{alignment: :nal},
```

Comment:
Unaliased for better readability in `StreamFormatError` output logs (with the alias, it showed only `%H264{...}` instead of `%Membrane.H264{...}`